cfme/services/catalogs/orchestration_template.py

```diff
@@ -237,7 +239,7 @@ class AddDialog(CFMENavigateStep):
     VIEW = AddDialogView
 
     def step(self):
-        self.view.configuration.item_select('Create Service Dialog from Orchestration Template')
+        cfg_btn("Create Service Dialog from Orchestration Template")
```

Comment:
It worked just fine before we made it pretty.  Navmazing and widgetastic were the regressions in this particular case as no one ever bothered to test the changes.  All I did was use the exact same structure as is in catalog_item.py which is working with the new navigation and already approved.  The way it was written above, each View class would have to initialize the same self.view.configuration. to work, which it never did, and really doesn't make sense to do it that way.  There is precedent for the way I did it, but I'm open to suggestions.